marc.py

```diff
@@ -521,15 +521,19 @@ class MARCExporter(object):
         },
     ]
 
-    SETTINGS = [
-        { "key": STORAGE_PROTOCOL,
-          "label": _("Storage service protocol"),
-          "description": _("Storage protocol to use for uploading generated MARC files. The service must already be configured under 'Storage Services'."),
-          "type": "select",
-          "options": [{ "key": ExternalIntegration.S3, "label": ExternalIntegration.S3 }],
-          "default": ExternalIntegration.S3,
-        },
-    ]
+    NO_MIRROR_INTEGRATION = u"NO_MIRROR"
+    SETTING = {
+        "key": "mirror_integration_id",
+        "label": _("MARC Mirror"),
+        "description": _("Storage protocol to use for uploading generated MARC files. The service must already be configured under 'Storage Services'."),
+        "type": "select",
+        "options" : [
+            dict(
+                key=NO_MIRROR_INTEGRATION,
+                label=_("None - Do not mirror MARC files")
+            )
+        ]
+    }
```

Comment:
I'm at the very beginning of this diff so maybe this will make sense later, but how do the mirror integrations get added to `options`?

I would expect the config here to look more like:

```
"type": "storage_integration",
"storage_purpose": "MARC",
```